### ISSUE TITLE
Read extra notes

### ISSUE THREAD

**Issue by LordVladneutral** [rbavishi/Habitican-Curse#25]
It would be great if the program could display and/or modify the extra notes for any given task. I don't think it would be hard to add either.

**Comment by rbavishi** [comment]
Yes it should be easy. Unfortunately I'm busy for a couple of weeks. Maybe you or someone else can take it up?

**Comment by LordVladneutral** [comment]
Challenge accepted. Seriously I don't have the slightest idea of code, let alone in python, but reading the existing code, I might be able to do it. That's how you learn.

**Comment by LordVladneutral** [comment]
Nope, I can't... Not now, not ever, I have other things to learn urgently and too few time...

After all that time, I still which for updates... Anyone here ?
